include/libopencm3/stm32/l0/rcc.h

```diff
@@ -595,7 +595,9 @@ enum rcc_periph_rst {
 
 	/* APB1 peripherals*/
 	RST_TIM2 = _REG_BIT(0x28, 0),
+	RST_TIM3 = _REG_BIT(0x28, 1),
 	RST_TIM6 = _REG_BIT(0x28, 4),
+	RST_TIM7 = _REG_BIT(0x28, 5),
```

Comment:
You also need to add them to the RCC_ enum (for enabling disabling) the SCC_ enum (if they have low power mode bits) and you need to add definitions for people using the registers directly.  Look for all uses of TIM6 for instance.